### ISSUE TITLE
How enable both sass (nwb-sass) and css-modules?

### ISSUE THREAD

**Issue by equinusocio** [insin/nwb#171]
I can't figure out how enable both sass preprocessor and css-module at the same time. With a standard webpack configuration i can do this through the loaders query, but here? Atm this is my configuration that enable only css-modules

``` js
module.exports = {
  type: 'react-app',
  webpack: {
    loaders: {
      css: {
        modules: true,
        localIdentName: '[local]__[hash:base64:5]'
      }
    },
    extractText: {
      allChunks: true
    }
  }
}
```

**Comment by insin** [comment]
I need to pull the documentation for this out into the README for each preprocessor plugin- it's currently hiding at the bottom here:

https://github.com/insin/nwb/blob/master/docs/Plugins.md#implementing-css-preprocessors

TL;DR: change the prop in your `webpack.loaders` config from `css` to `sass-css`.

**Comment by equinusocio** [comment]
Thanks!

**Comment by malixsys** [comment]
```
loaders: {
      'sass-css': {
        modules: true,
        localIdentName: '[name]__[local]__[hash:base64:5]'
      }
    }
```
